tests/test_packages/test_skills/test_simple_aggregation/test_behaviours.py

```diff
@@ -152,31 +160,22 @@ def test_setup(self):
         mock_logger.assert_any_call(
             logging.INFO, "registering agent on SOEF.",
         )
-        mock_logger.assert_any_call(
-            logging.INFO, "registering service on SOEF.",
-        )
 
-        descriptions = [
-            self.aggregation_strategy.get_location_description(),
-            self.aggregation_strategy.get_register_service_description(),
-            self.aggregation_strategy.get_register_personality_description(),
-            self.aggregation_strategy.get_register_classification_description(),
```

Comment:
check what happened to those

Reply:
These are for the service registration which don't happen in setup anymore. They are called from the handler after getting SUCCESS. The method's tests are below see `test_register_service_personality_classification`.

Reply:
Are they called sequentially? (They should be)

Reply:
Yes (they are mocked now)